packages/docs/PropTables/FileUploaderPropTable.tsx

```diff
@@ -33,8 +54,14 @@ const fileUploaderProps: Prop[] = [
   },
   {
     name: 'dropzoneConfig',
-    types: '{ label?: string; description?: string; icon?: ReactNode }',
-    description: 'Adds a label and a description to the drop-zone box.',
+    types: '{ label?: string; description?: string; icon?: ReactNode, action: object }',
```

Comment:
At this point we probably should split this into its own tab on the prop table.

Reply:
I have extracted it to a separate table.